package.json

```diff
@@ -7,7 +7,7 @@
     "husky": "^1.3.1",
     "jasmine": "^3.5.0",
     "tslint": "^5.9.1",
-    "typescript": "^3.8.3"
+    "typescript": "^4"
```

Comment:
Could you clarify why we need TS 4?

Reply:
Is it required in order to use ES2020?

Reply:
[This `tsc` bug](https://github.com/microsoft/TypeScript/pull/38013) was merged to master on June 18th.  Typescript 3.9 was [released May 12th](https://devblogs.microsoft.com/typescript/announcing-typescript-3-9/) and Typescript 4/0 was released [August 12th](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/).  So we need Typescript 4.0 in order to get that bugfix.  Alternatively we could cast all of our uses of `Intl.NumberFormatter` but that seems like a worse idea than just upgrading our Typescript version.